cdi-unit/src/main/java/org/jglue/cdiunit/internal/TestConfiguration.java

```diff
@@ -0,0 +1,55 @@
+package org.jglue.cdiunit.internal;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Defines, how the WeldTestUrlDeployment should initialize Weld.
+ *
+ * @author aschoerk
+ */
+public class TestConfiguration {
+
+    public TestConfiguration(Class<?> testClass, Method testMethod, Collection<Class<?>> additionalClasses) {
+        this.testClass = testClass;
+        this.testMethod = testMethod;
+        this.additionalClasses = additionalClasses != null ? additionalClasses : new ArrayList<>();
```

Comment:
Instead of allowing null for additionalClasses, I think it would be better to use an empty collection, and mark the parameter NonNull (or use ParametersAreNonnullByDefault). I suppose we need com.google.code.findbugs:jsr305 on the classpath (scope: provided).